arangod/Aql/ExecutionBlockImpl.tpp

```diff
@@ -1049,6 +1049,32 @@ auto ExecutionBlockImpl<Executor>::executeFetcher(ExecutionContext& ctx,
                 return;
               }
 
+              // We are entering debugging land here.
+              // Someone hasasked as to stop async tasks.
+              // But here we are just starting to work on one.
+              // This is okay and can rarely happen. But if it happens
+              // We want to know if we start the task and finish it quickly
+
+              bool hasStoppedAsyncTasks = block->hasStoppedAsyncTasks();
+              if (hasStoppedAsyncTasks) {
+                LOG_TOPIC("14d22", WARN, Logger::AQL)
+                    << "[query#" << block->getQuery().id() << "] ALERT"
+                    << block->printBlockInfo()
+                    << " was asked to stop async task. We still start one. "
+                       "This is an allowed rare race.";
+              }
+
+              auto stopGuard =
+                  ScopeGuard([block, hasStoppedAsyncTasks]() noexcept {
+                    if (hasStoppedAsyncTasks) {
+                      LOG_TOPIC("14d21", WARN, Logger::AQL)
+                          << "[query#" << block->getQuery().id()
+                          << "] CLEAR ALERT" << block->printBlockInfo()
+                          << " We completed the task of the aforementioned "
+                             "race. All is fine.";
+                    }
+                  });
```

Comment:
These should not be on `WARN`, but probably on `DEBUG`.

Reply:
If they are DEBUG, then we do not see them in production, or?

Reply:
Unless we increase the log level on deployments of interest. But these aren't messages we generally want to see on WARN, or am I wrong?